### PR TITLE
support development Go version

### LINKED ISSUE #78: Support to parse devel version
Recently, skeleton changed the logic from runtime.Version to use go env GOVERSION to decide the Go version.
But, this logic might be failed if the Go version is development version.
Actually, I ran the latest skeleton and received the error like this.

```
Error: template: skeleton:59:2: executing "skeleton" at <gomodinit .Path>: error calling gomodinit: create go.mod: invalid language version "devel go1.24-f99f5da18f Thu Nov 14 22:29:26 2024 +0000"
```

So, I think skeleton should consider the users that uses development Go.

### PR DESCRIPTION
**Pull request by sivchari**
close #78 
I supported development Go version.

In additional, I filed the patches to [skeletonkit](https://github.com/gostaticanalysis/skeletonkit/pull/11), too. When [it](https://github.com/gostaticanalysis/skeletonkit/pull/11) will be merged,  I upgrade the dependency of skeletonkit.

### CODE REVIEW

File: v2/skeleton/skeleton.go [hunk 1]
```diff
@@ -45,6 +45,12 @@ func Main(version string, args []string) int {
 		return ExitError
 	}
 
+	if strings.HasPrefix(gover, "devel ") {
+		// The devel format is like following.
+		// devel go1.24-xxxxxxxxxx Day Date Mon Time Arch"
+		gover = strings.Split(strings.TrimPrefix(gover, "devel "), "-")[0]
```
Comment (tenntenn) [inline v2/skeleton/skeleton.go:51]: Could you use `strings.Cut` instead of `strings.Split`?

**Review by tenntenn — CHANGES_REQUESTED** [review]
I left a comment.

File: v2/skeleton/skeleton.go (hunk 1 above)
Comment (sivchari) [reply v2/skeleton/skeleton.go]: It's reasonable, thanks. I fixed it.

**Review by tenntenn — APPROVED** [review]
LGTM

**Comment by sivchari** [comment]
Hi, @tenntenn 
This problem is still existing because the patch that I filed to skeletonkit hasn't been merged yet.
So we should manage it. Thanks.

**Pull request merged**